### PR TITLE
snapcraft: Add snap packaging

### PR DESCRIPTION
**Pull request by stgraber**
Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>

### CODE REVIEW

**Comment by stgraber** [comment]
A current build of this is available in the snap store, so one can do `snap install nsd-exporter`, then just copy the certificates from `/etc/nsd` over to `/var/snap/nsd-exporter/current/` so the snap can access them.

If this gets merged, the upstream maintainer may want to take over the snap in the store and setup automated builds so new commits on github result in a new snap package available in edge channel in the store.

File: .snapcraft/daemon.start
Comment (optix2000) [inline .snapcraft/daemon.start:12]: I'm not familiar with snap, how do you document to the user that the default listen address is different on snap?

**Review by optix2000 — APPROVED** [review]
Will need to look into snap some more before merging, but at a high level seems reasonable.

**Pull request closed without merging**
